data/json/effects.json

```diff
@@ -1458,15 +1458,23 @@
   {
     "type": "effect_type",
     "id": "hypocalcemia",
-    "name": [ "Hypocalcemia", "Weak bones", "Brittle bones" ],
-    "desc": [ "A lack of calcium in your diet will make your bones progressively weaker." ],
-    "apply_message": "Your bones are becoming more brittle.",
-    "remove_message": "Your bones regain their usual strength.",
+    "name": [ "Mild Hypocalcemia", "Hypocalcemia", "Acute Hypocalcemia" ],
+    "desc": [ "A lack of calcium in your diet will cause various neuromuscular symptoms." ],
+    "apply_message": "Your muscles are becoming more stiff.",
+    "remove_message": "Stiffness in your musles passes away.",
     "decay_messages": [
       [ "Your calcium deficiency is nearly resolved.", "good" ],
-      [ "Your bones become stronger as your calcium deficiency improves.", "good" ]
+      [ "Your have better control of your muscles as your calcium deficiency improves.", "good" ]
     ],
     "max_intensity": 3,
+    "base_mods": {
+      "dex_mod": [ -1 ],
+      "speed_mod": [ -10 ]
+    },
+    "scaling_mods": {
+      "dex_mod": [ -2 ],
+      "speed_mod": [ -15 ]
```

Comment:
Really, hypocalcaemia in a gameplay relevant way has nothing to do with bones. You'd have to be severely hypocalcaemic for many years to get osteoporosis.

In game, hypocalcaemia that matters would be profound, nerve-affecting lack of calcium. It is very hard to get, but the main symptoms would be numbness and seizures, followed by death if it hits too low a level.

For sanity checking, this is not something you can really get through malnutrition unless you try *extremely* hard. I am not convinced it belongs in the game at all yet.